_posts/en/newsletters/2024-12-13-newsletter.md

```diff
@@ -0,0 +1,308 @@
+- **Java-based HWI released:**
+  [Lark App][larkapp github] is a command line application for interacting with
+  hardware signing devices. It uses the [Lark Java library][lark github], a port
+  of [HWI][topic hwi] for the Java programming language.
+
+- **Saving Satoshi Bitcoin development education game announced:**
+  The [Saving Satoshi][saving satoshi website] website provides interactive
+  educational exercises for those new to Bitcoin development.
+
+- **Neovim Bitcoin Script plugin:**
+  The [Bitcoin script hints][bsh github] Neovim plugin for Rust shows Bitcoin
+  script stack states for each operation within the editor.
+
+- **Proton Wallet adds RBF:**
+  Proton Wallet users can now [RBF][topic rbf] fee bump [their
+  transactions][proton blog].
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+- [Core Lightning 24.11rc3][] is a release candidate for the next major
+  version of this popular LN implementation.
+
+- [LND 0.18.4-beta.rc1][] is a release candidate for a minor version of
+  this popular LN implementation.
+
+- [Bitcoin Core 28.1RC1][] is a release candidate for a maintenance
+  version of the predominant full node implementation.
+
+- BTCPay FIXME:harding
+
+<!-- FIXME:harding to update section on Thursday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #31096][] Package validation: accept packages of size 1
+
+- [Bitcoin Core #31175][] rpc: Remove submitblock pre-checks
+
+- [Bitcoin Core #31112][] Improve parallel script validation error debug logging
+
+- [LDK #3446][] Support Trampoline flag in BOLT12 invoices
+
+- [Rust Bitcoin #3682][] Add API scripts and output files
+
+- [BTCPay Server #5743][] Multisig/watchonly wallet transaction creation flow proof of concept
```

Comment:
```suggestion
- [BTCPay Server #5743][] Multisig/watchonly wallet _transaction creation flow_ proof of concept
```

Reply:
FYI: this was placeholder text sourced from the commit message.  It gets replaced by @Gustavojfe's write up, so it doesn't need editing.